pyproject.toml

```diff
@@ -150,6 +150,18 @@ version = { attr = "integreat_compass.__version__" }
 [tool.setuptools.packages.find]
 include = ["integreat_compass*"]
 
+[tool.bumpver]
+current_version = "2023.0.0"
+version_pattern = "YYYY.MM.INC0[-TAG]"
+commit_message = """
+Bump version to {new_version}
+
+[tool.bumpver.file_patterns]
+"pyproject.toml" = [
+    'current_version = "{version}"',
+    'version = "{version}"',
```

Comment:
```suggestion
```

This pattern does not exist in `pyproject.toml`. Instead, the version is dynamically defined here (so the `__init__.py` needs to be listed in `file_patterns`):
https://github.com/digitalfabrik/integreat-compass/blob/75b3e26d507f828d6cde4206a76fdc10f8b58633/integreat_compass/__init__.py#L6

Reply:
Oh thank you, that explains a lot!

Reply:
This is not yet resolved. The version in `__init__.py` is still missing from the file patterns.